test/integration/globals/app-nav.js

```diff
@@ -10,7 +11,11 @@ context('App Nav', function() {
           last_name: 'McTester',
         };
 
-        fx.data.relationships.organization.data.id = '1';
+        _.each(fx.included, inc => {
```

Comment:
Rather than generating and getting a collection of organizations right now, why don't we make one and put it in `tests/` with a `11111` id.. then we won't need to worry about that org being awkward